src/cmd/device-protection.js

```diff
@@ -109,8 +109,10 @@ module.exports = class DeviceProtectionCommands extends CLICommandBase {
 			} catch (error) {
 				if (error.message === 'Not supported') {
 					throw new Error(`Device Protection feature is not supported on this device. Visit ${chalk.yellow('https://docs.particle.io')} for more information${os.EOL}`);
+				} else if (error.message === 'Device public key was not found') {
```

Comment:
Can you move this inside `DeviceProtectionHelper.disableDeviceProtection` so that any command that disables device protection will get this behavior.